example/spec/parallel/discoverGranules/DiscoverGranulesHttpsSpec.js

```diff
@@ -36,10 +37,12 @@ xdescribe('The Discover Granules workflow with https Protocol', () => {
 
     const providerJson = JSON.parse(fs.readFileSync(`${providersDir}/https_provider.json`, 'utf8'));
 
-    const provider = {
-      ...providerJson,
+    // we actually want https for this test. we will later update provider to use https
+    const provider = Object.assign(providerJson, {
+      protocol: 'http',
+      host: getProviderHost(providerJson),
```

Comment:
Is there a way to set the provider port as well?  This will always default to port 80.

Reply:
It's not defaulting to port 80, it's defaulting to whatever is configured in the provider.